testpilot/frontend/static-src/app/views/experiment-list-view.js

```diff
@@ -1,19 +1,32 @@
 import app from 'ampersand-app';
 import BaseView from './base-view';
 import ExperimentRowView from './experiment-row-view';
-
+import ExperimentsCollection from '../collections/experiments';
 
 export default BaseView.extend({
   template: `<div class="card-list experiments"></div>`,
 
   props: {
-    hasAddon: 'boolean'
+    hasAddon: 'boolean',
+    except: { type: 'string', required: false }
   },
 
   render() {
     BaseView.prototype.render.apply(this, arguments);
+    console.log('experiments', app.experiments);
+
+    const filteredExperiments = () => {
+      if (this.except) {
+        const filtered = app.experiments.models.filter(experiment => {
+          return experiment.slug !== this.except;
+        });
+        return new ExperimentsCollection(filtered);
+      }
```

Comment:
Looking over ampersand-filtered-subcollection and it looks like overkill for what we need anyways. I think what you've got here is rather clear.